22.12 updates (#26)
CTSRD-CHERI/cheribsd-getting-started: src/features/README.md

both ABIs, although CheriABI packages are currently considered experimental.
This is discussed further in the [chapter on packages](../packages/).

## CheriABI desktop environment (experimental)

As of the 22.12 release, the installer has gained the option to install a

> desktop environment using the panfrost GPU on the Morello System-on-Chip.

Reviewer: ```suggestion
desktop environment using the Panfrost GPU on the Morello System-on-Chip.
```

The option installs a basic desktop environment using KDE and Wayland

> compiled for CheriABI with the `cheribsd-desktop` package.  It also
> installs a hybrid Firefox web browser via the `cheribsd-desktop-hybrid-extras`

Reviewer 2: ```suggestion
compiled for CheriABI with the `cheri-desktop` package.  It also
installs a hybrid ABI Firefox web browser via the `cheri-desktop-hybrid-extras`
```

## Software compartmentalization (experimental)

CheriBSD implements, two different
CHERI-enabled software compartmentalization models:

- **Colocated processes (co-processes) compartmentalization** accelerates
  UNIX Inter-Process Communication (IPC) and context switching by colocating
  multiple processes in the same address space, separating them using CHERI
  capabilities.  Support for co-processes is maintained in an experimental

>   development branch and is not included in current software releases.

Author: @trasz We might want to point people somewhere for more information.

- **Dynamic-linker-based compartmentalization** isolates shared libraries
  within a process using CHERI capabilities limiting the access of attackers
  who have achieved arbitrary code execution within a library.
  Initial support for linker-based library compartmentalization is included

>   in the 22.12 release of CheriBSD.

Author: @dpgao We should probably point people somewhere for more information.

## Software compartmentalization (experimental)

> CheriBSD implements, two different

Reviewer: ```suggestion
CheriBSD implements two different
```
